Dockerfile

```diff
@@ -58,7 +58,7 @@ RUN apt-get update -q && apt-get install -y \
 RUN apt-get update -q \
     && apt-get install -y apt-transport-https ca-certificates \
     && curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - \
-    && curl https://packages.microsoft.com/config/ubuntu/20.04/prod.list > /etc/apt/sources.list.d/mssql-release.list \
+    && curl https://packages.microsoft.com/config/ubuntu/22.04/prod.list > /etc/apt/sources.list.d/mssql-release.list \
```

Comment:
@guruofgentoo can we drop ODBC support?

Reply:
Looks like @guruofgentoo already [answered this question](https://github.com/level12/python-test-multi/issues/13) w/ a "no."

Reply:
We don't have current projects using SQL Server. But, keg has some stuff for the mssql dialect, and webgrid runs tests with it too. If we drop it here, we should move this setup directly into the CI runs for those libraries.

Reply:
I don't mind keeping it.  Just didn't want to if there was no reason.